.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,9 @@
 <!--- 
 Replace this comment with a description of the problem.
 
-Please help us by providing all of the details listed below. A complete and thoughtful issue request with research on possible root causes and diagnostics goes a very long way in helping us rapidly get your issues resolved. 
+Please help us by providing all of the details listed below. A complete and thoughtful issue request with research on possible root causes and diagnostics goes a very long way in helping us rapidly get your issues resolved.
```

Comment:
maybe we could add a title to this section as well

like 
**description of the problem:**

BTW I prefer formatting of PR template with the kind:
### What is the problem?

Reply:
fixed